signer/core/api_test.go

```diff
@@ -99,12 +99,16 @@ func (ui *headlessUi) ApproveNewAccount(request *core.NewAccountRequest) (core.N
 
 func (ui *headlessUi) ShowError(message string) {
 	//stdout is used by communication
-	fmt.Fprintln(os.Stderr, message)
+	if err := fmt.Fprintln(os.Stderr, message); err != nil {
```

Comment:
There is no need to check for errors when writing to stdout or stderr. Please remove.